phpBB/styles/prosilver/theme/content.css

```diff
@@ -88,6 +88,18 @@ ul.topiclist dfn {
 	padding-top: 5px;
 }
 
+.forum-desc {
+	display: none;
+}
+
+.forum-mods {
+	display: none;
+}
+
+.forum-subs {
+	display: none;
+}
+
```

Comment:
Why are you hiding this information?

Reply:
Sorry it was just a test it should be block I'll fix

Reply:
And yes it's related to that topic